CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 2.5.2 (2020-09-04)
+
+### Breaking changes
```

Comment:
when you have breaking changes, you should increment minor version: 2.6.0

That being said, I don't think we want that change to be publish without the new Input module to be implemented.